src/v1/sources/shopify/webhookTransformations/serverSideTransform.js

```diff
@@ -77,6 +77,32 @@ const trackPayloadBuilder = (event, shopifyTopic) => {
   return message;
 };
 
+/**
+ * Creates an identify event with userId and anonymousId from message object and identifyMappingJSON
+ * @param {String} message
+ * @returns {Message} identifyEvent
+ */
+const createIdentifyEvent = (message) => {
+  const { userId, anonymousId, traits } = message;
+  const identifyEvent = new Message(INTEGERATION);
+  identifyEvent.setEventType(EventType.IDENTIFY);
+  identifyEvent.userId = userId;
+  identifyEvent.anonymousId = anonymousId;
+  const mappedTraits = {};
```

Comment:
Should we also set context.traits?

Reply:
As per product confirmation, doing the same.